docs/execution-providers/DirectML-ExecutionProvider.md

```diff
@@ -11,11 +11,12 @@ redirect_from: /docs/reference/execution-providers/DirectML-ExecutionProvider
 
 The DirectML Execution Provider is a component of ONNX Runtime that uses [DirectML](https://docs.microsoft.com/en-us/windows/ai/directml/dml-intro) to accelerate inference of ONNX models. The DirectML execution provider is capable of greatly improving evaluation time of models using commodity GPU hardware, without sacrificing broad hardware support or requiring vendor-specific extensions to be installed.
 
+
 DirectML is a high-performance, hardware-accelerated DirectX 12 library for machine learning on Windows.  DirectML provides GPU acceleration for common machine learning tasks across a broad range of supported hardware and drivers.
 
 When used standalone, the DirectML API is a low-level DirectX 12 library and is suitable for high-performance, low-latency applications such as frameworks, games, and other real-time applications. The seamless interoperability of DirectML with Direct3D 12 as well as its low overhead and conformance across hardware makes DirectML ideal for accelerating machine learning when both high performance is desired, and the reliability and predictability of results across hardware is critical.
 
-The DirectML Execution Provider currently uses DirectML version [1.8.2](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.8.2) and supports up to ONNX opset 12 ([ONNX v1.7](https://github.com/onnx/onnx/releases/tag/v1.7.0)). Evaluating models which require a higher opset version is unsupported and will yield poor performance.
+The DirectML Execution Provider currently uses DirectML version [1.8.0](https://www.nuget.org/packages/Microsoft.AI.DirectML/1.8.0) and supports up to ONNX opset 12 ([ONNX v1.7](https://github.com/onnx/onnx/releases/tag/v1.7.0)). Evaluating models which require a higher opset version is unsupported and will yield poor performance.
```

Comment:
Since we're updating the version anyway (though, why backwards to 1.8.0?), we should update it to 1.9.0, which is what it currently points to.

Reply:
Oh that might have been an artifact of the version of the files I used to create the PR. I will fix